breed/index.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>About Doodle Dandy | Doodle Dandy</title>
+    <link rel="stylesheet" href="../assets/vendor/simplecss/simple.min.css" />
+    <link
+      rel="stylesheet"
+      href="../assets/vendor/fontawesome/6.2.1/css/fontawesome.min.css"
+    />
+    <link
+      rel="stylesheet"
+      href="../assets/vendor/fontawesome/6.2.1/css/brands.min.css"
+    />
+    <link
+      rel="stylesheet"
+      href="../assets/vendor/fontawesome/6.2.1/css/solid.min.css"
+    />
+    <link rel="stylesheet" href="../assets/styles/main.css" />
+    <link
+      rel="shortcut icon"
+      href="../assets/images/favicon.ico"
+      type="image/x-icon"
+    />
+  </head>
+
+  <body>
+    <header>
+      <img src="../assets/images/header.png" alt="Doodle Dandy" />
+      <nav>
+        <a href="/">Home</a>
+        <a href="/contact">Contact us</a>
+        <a href="/about">About us</a>
+        <a href="/breed" class="current">Breed Info</a>
+      </nav>
+    </header>
+    <main>
+      <h1>AussieDoodle Breed Info</h1>
+
+      <p>
+        The Aussiedoodle is a delightful hybrid breed created by crossing a
+        Poodle with an Australian Shepherd. This charming mix combines the best
+        traits of both parent breeds, resulting in a dog that is not only
+        incredibly adorable but also highly intelligent and playful.
+        <img
+          src="../assets/images/ddpup2.jpg"
+          alt="Doodle Dandy Puppy Love"
+          class="dog"
+        />
```

Comment:
Images should not be inside paragraph elements. You should do separate p elements for each of the text sections and keep the images outside of them. You'll get rid of the br elements as well.